backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlTableFieldFactory.java

```diff
@@ -826,7 +827,10 @@ private DataFetcher fetcher(Schema schema, MutationType mutationType) {
               result.append("updated " + count + " records to " + tableName + "\n");
               break;
             case INSERT:
-              count = table.insert(TypeUtils.convertToRows(table.getMetadata(), rowsAslistOfMaps));
+              count =
+                  table.insert(
+                      TypeUtils.convertToRows(table.getMetadata(), rowsAslistOfMaps),
+                      JavaScriptBindings.simplePostClient);
```

Comment:
also for update then :-)
But lets think why this would be bad idea :-)

I think the main thing is that simplePostClient should be using a graphql instance that is of same user as the current user. Will check code later if that indeed is the case.